dockstore-integration-testing/src/test/java/io/dockstore/client/cli/WorkflowIT.java

```diff
@@ -234,14 +234,75 @@ public void testWorkflowLaunchOrNotLaunchBasedOnCredentials() throws IOException
 
         // should be able to launch properly with correct credentials even though the workflow is not published
         FileUtils.writeStringToFile(new File("md5sum.input"), "foo" , StandardCharsets.UTF_8);
-        systemExit.expectSystemExitWithStatus(0);
```

Comment:
Was this deletion intentional?

Reply:
Yes, the exit code of 0 is implicit if there is no failure already.